packages/binding-http/src/http-server.ts

```diff
@@ -311,9 +312,9 @@ export default class HttpServer implements ProtocolServer {
                     ];
                 }
                 if (!thing.forms) {
-                    thing.forms = [];
+                    thing.forms = new Array() as [FormElementRoot];
```

Comment:
alternatively : 
```` 
= [] as FormElementRoot[];
```` 
or
````
= < FormElementRoot[]>[];
````

Reply:
neither  `[] as FormElementRoot[]` nor `< FormElementRoot[]>[];` is working

`[] as [FormElementRoot]` does not work either.
Hence  I suggest to keep it as is till we have decided what to do with proper `form` typing.